src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,44 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
you can use destructuring here
but dont forget to change => () to {} and ### return ( html )